sdks/python/apache_beam/transforms/util.py

```diff
@@ -51,6 +51,7 @@
 from apache_beam.transforms.core import MapTuple
 from apache_beam.transforms.core import ParDo
 from apache_beam.transforms.core import Windowing
+from apache_beam.transforms.core import _fn_takes_side_inputs
```

Comment:
Perhaps we should move `_fn_takes_side_inputs` to util.py now it is used in multiple places and rename it to `fn_takes_side_inputs`?